### PR TITLE
update contributors

### PR DESCRIPTION
**Pull request by TonyAble**

### CODE REVIEW

**Comment by yxliang01** [comment]
Hmm. I had a PR #353 being merged. It was a mirror README fix, unsure whether it's enough to be added as contributor. But since you filed this PR for #357 which has similar nature, can you also help me add to the list without having too many PRs, so that they can be merged altogether? Thanks, @TonyAble !

**Comment by TonyAble** [comment]
> Hmm. I had a PR #353 being merged. It was a mirror README fix, unsure whether it's enough to be added as contributor. But since you filed this PR for #357 which has similar nature, can you also help me add to the list without having too many PRs, so that they can be merged altogether? Thanks, @TonyAble !

OK,I'll try

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Open Chinese Convert (OpenCC, 開放中文轉換) is an opensource project for c
 
 * [Debian](http://packages.qa.debian.org/o/opencc.html)
 * [Ubuntu](https://launchpad.net/ubuntu/+source/opencc)
-* [Fedora](https://admin.fedoraproject.org/pkgdb/package/opencc/)
+* [Fedora](https://apps.fedoraproject.org/packages/opencc/)
```
Comment (lotem) [inline README.md]: this change has already happened in master. may be your code is out of date.

File: README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ See [android-opencc](https://github.com/qichuan/android-opencc)
 
 * [ibus-pinyin](https://github.com/ibus/ibus-pinyin)
 * [fcitx](https://github.com/fcitx/fcitx)
-* [rimeime](http://code.google.com/p/rimeime/)
+* [rimeime](https://rime.im/)
```
Comment (lotem) [inline README.md]: this change is already in master

**Comment by lotem** [comment]
The changes seem to be out of date.
Please sync your git client or withdraw the pull request.

**Comment by lotem** [comment]
Thanks.
You are still welcome to the list of contributors once the conflicting changes in the PR branch are resolved.

**Pull request closed without merging**